charts/dashboard/values.yaml

```diff
@@ -4,7 +4,7 @@ global:
 dashboard:
   dashboard:
     image:
-      repository: docker.io/projectsveltos/dashboard
+      repository: mirror.gcr.io/projectsveltos/dashboard
```

Comment:
thank you for proposing this. can we do this via an helm value? Leaving docker.io the default one. 
If set switch to mirror.gcr

Reply:
In my studies, I have been wondering why we continue to specify docker.io. Considering that mirror.gcr.io is a complete pull-through cache for docker.io and does not enforce Google's rate limits, why do we default to using docker.io? Currently, even just launching sveltos twice using `make quickstart` on a local kind cluster causes us to hit docker.io's rate limits, which I find to be quite a significant pitfall.

Reply:
Thank you @kahirokunn I agree with what you said. Yet I'm afraid of changing default behavior because I cannot be sure nobody else will be affected. Let me push a change where we allow using mirror.gcr as a knob.